setup.py

```diff
@@ -93,11 +93,7 @@ def find_meta(meta):
         packages=find_packages(where="src"),
         package_dir={"": "src"},
         install_requires=[
-<<<<<<< HEAD
```

Comment:
@julianz- I see you may be struggling with Git a little. I'll try to help.

If you want to melt the last commit into the previous one, do the following:
```console
$ git rebase -i HEAD^^
```
That will open an editor (for me it's NeoVim, you can have something different set up, like VS Code).
In the opened text file, in the "rebase plan", find the last entry with the commit 9c22966.
On that line, change the first word "pick" to just one letter "f".
Save the change and close the editor.
At this point, you'll end up with the commit you want, and you can force-push that.

Reply:
I just separated out some code into a different PR which was approved. But I am now having difficulty in rebasing this PR so that it doesn't include that code change. The rebased version keeps saying it's one commit behind the upstream main version even though I rebased against that version. I did 

```git fetch --all```

as you advised previously for this kind of circumstance - and which worked before.

Reply:
@webknjaz Ok I got past that problem (not sure why but using  git merge-base jan23-2024 upstream/main was not giving me the right result) but now the code coverage is failing even though I already dealt with the block of code that never runs because of the cryptography version on the previous PR. Let me know what you think. Thanks!

Reply:
@julianz- the coverage report page (https://app.codecov.io/gh/pyca/pyopenssl/pull/1287/indirect-changes#1beeb2c0c32f16d580bcd3b7da99eecc-R229) indicates an indirect coverage change @ https://github.com/pyca/pyopenssl/blob/482b95e/src/OpenSSL/SSL.py#L230.
However, that line isn't covered on `main`. So maybe, another rebase will clear that confusion. If it doesn't — ignore it.

Reply:
It's showing up because `OP_LEGACY_SERVER_CONNECT` always exists with the new, higher, cryptography minimum version.

You can remove the `except AttributeError`

Reply:
Most likely, the underlying OpenSSL lib used in tests got upgraded to a version that always provides that constant. cc @mhils

Reply:
> You can remove the `except AttributeError`

Ah, makes sense. I wasn't entirely sure. I think the confusion of `main` vs. `master` threw me off to make an incorrect assumption. Seems like you're right and it is indeed related to the change in this PR.

Reply:
@alex @webknjaz I removed the check on OP_LEGACY_SERVER_CONNECT. That's seems to have cleared the coverage problem. Thx!